config/config.yaml

```diff
@@ -36,3 +36,11 @@ queryConfig:
   serverConfig:
     serverPort: ${KALDB_QUERY_SERVER_PORT:-8081}
     serverAddress: ${KALDB_QUERY_SERVER_ADDRESS:-localhost}
+
+metadataStoreConfig:
+  zookeeperConfig:
+    zkConnectString: "127.0.0.1:2181"
+    zkPathPrefix: "kaldb"
+    zkSessionTimeoutMs: 1000
```

Comment:
Ideal ZK session timeout values are: 
minSessionTimeout = ticktime * 2,  maxSessionTimeout=ticktime * 20.

Also, the session timeout should be higher than connection timeout. If we don't know the ZK tick time, a placeholder ZK session timeout to be 5 secs for now. Also, the connection timeout of 1.5 sec seems high. 500ms may be more appropriate with 100ms between reconnects.

Reply:
Originally I just copied over the example from the tests, but then I forgot to go back and edit these. I set the values you suggested, and made these all settable via ENV vars as well.